packages/react-vapor/src/components/tooltip/examples/TooltipExamples.1.use.md

```diff
@@ -0,0 +1,19 @@
+## Guidelines
+
+### Do
+
+-   Keep the **text short** and direct (3 lines maximum). For more detailed explanations, link to external documentation.
+-   Make Tooltips **appear in empty areas** in the screen, or above less relevant information.
+-   Only show **one Tooltip at a time**.
+
+### Do Not
+
+-   Do not use Tooltips to display **information that most users need**. Use regular text instead.
+-   Avoid adding Tooltips that display the same text as what **can already be read in the interface** (e.g., a Tooltip with "Print" as its content is not necessary on a button with the label “Print”).
+-   Avoid using Tooltips for information regarding **errors or warnings**. Displaying this information is too important to depend on hovering.
+
+---
+
+## Related Components
+
+[Popover](/#/components/PopoverConnected)
```

Comment:
The links should always start with a `#`, otherwise they might break depending on where we deploy (if you try your demo you'll see what I mean)

```suggestion
[Popover](#/components/PopoverConnected)
```